sapl/templates/relatorios/blocos_sessao_plenaria/oradores_expediente.html

```diff
@@ -2,6 +2,7 @@ <h2 class="gray-title">Oradores do Expediente</h2>
 
     {% for orador in lst_oradores_expediente%}
         <tr>
-            <p> <b>{{orador.num_ordem}}</b> - {{orador.nom_parlamentar}}/{{orador.sgl_partido}}</p>
+            <p> <b>{{orador.num_ordem}}</b> - {{orador.nom_parlamentar}}/{{orador.sgl_partido}}
+                {% if orador.observacao %} ({{orador.observacao}}){% endif %}</p>
```

Comment:
Inserir uma quebra de linha. Se a observação for um pouco grandinha vai ficar estranho. Tô pensando em algo assim:
```
1. Fulano da Silva
    Destacou que as matérias ora postas em votação não tem relevância para a situação atual na qual
 se encontra o município, pois apenas discutem mudanças na legislação vigente sem sequer uma que
 faça requerimento ao prefeito.
2. Maria Cicrana
3. Beltrano da Silva
   Recusou-se a sair do plenário mesmo após finda a sessão. 
```